### DISCUSSION TITLE
[Ideia e Revisão para o Site] - Grupos de Whatsapp e Formas de Contato

### DISCUSSION THREAD

**Discussion by endersonmenezes** [codaqui/institucional#67] [Ideias]
O Grupo de Whatsapp, o Discord e o Workadventure, está espalhado pelo site. Vamos ver onde ele está faltando e se algum lugar está errado?

@ArtumosGOC @LuizFernando-TC @thaifurforo @SOPHI-A9

**Comment by LuizFernando-TC** [comment]
Na aba 'Contato' precisa de uma mudança. Pode ser adicionado esses links na aba 'Contato'.
Acho legal na aba 'Contato' ter uma sub aba escrito 'Comunidade' com os links para entrar para a comunidade (esses três links).
A sub aba 'Contato' poderia ter uma modificação nos botões com links, criando um quadro maior com os links.
Seria legal também adicionar gif's para mostrar como entrar na comunidade. O que acha?

**Reply by endersonmenezes** [reply]
Não entendi a parte dos gifs, me ajuda exemplificar que a @thaifurforo vai ajudar a gente =]

**Reply by LuizFernando-TC** [reply]
Bom, os gif's seriam para mostrar como se inscrever para participar do projeto, seguir a Codaqui nas redes sociais e etc. Talvez não seja muito necessário. Talvez um tutorial escrito já resolve, ou nem isso precise.

**Reply by endersonmenezes** [reply]
@thaifurforo quero sua opinião kkkk

**Comment by LuizFernando-TC** [comment]
Na aba 'inicio', o link do discord está dentro do tópico "Trilhas de aprendizado", poderia ter um tópico escrito "Comunidade" com esses três links, tirando o link do discord do tópico "Trilhas de aprendizado". O que acha?

**Reply by endersonmenezes** [reply]
Acho perfeito! Quer fazer esse caso também, ou eu faço, você quem manda o/

**Reply by LuizFernando-TC** [reply]
Como você iniciou a discussão, pode fazer, sua preferência.

**Comment by endersonmenezes** [comment]
Então ficaria:

*Página Contato*:
- Inserir os links que faltam para acesso a comunidade.
- Gifs mostrando como entrar em contato.
  - Exemplo 1

*Página Inicio*:
- Link do Discord separado por que não é uma trilha.
- Tópico Comunidade com essas descrições;

**Reply by LuizFernando-TC** [reply]
Isso, a aba contatos ficará um pouco vazia mas tudo bem, digo, sem muito conteúdo, mas é assim mesmo.

**Comment by endersonmenezes** [comment]
Issue criada: https://github.com/codaqui/institucional/issues/77
